kiali-operator/values.yaml

```diff
@@ -37,6 +37,11 @@ watchNamespace: ""
 # Note that this will be overriden to "true" if cr.create is true and cr.spec.deployment.accessible_namespaces is ['**'].
 clusterRoleCreator: true
 
+# Set to a list of secrets if you want the operator to be able to read those secrets across the cluster. This is necessary
+# if you want to support Kiali CRs with spec.kiali_feature_flags.certificates_information_indicators.enabled = true.
+# If you set this to an empty list, the operator will not be given permission to read secrets outside of the control plane namespace.
```

Comment:
I would like to reword this now that I'm looking at it and reading what I wrote. That last sentence isn't accurate because the operator will still have permission to read a secret named "kiali-signing-key" found anywhere. And it isn't true that the operator can't read secrets "outside of the control plane namespace' because the operator doesn't know ahead of time what control  plane namespaces there are or will be. So let's reword this to make it more clear and specific as to what this setting does:

```suggestion
# Set to a list of secrets in the cluster that the operator will be allowed to read. This is necessary if you want to
# support Kiali CRs with spec.kiali_feature_flags.certificates_information_indicators.enabled=true.
# The secrets in this list will be the only ones allowed to be specified in any Kiali CR (in the setting
# spec.kiali_feature_flags.certificates_information_indicators.secrets).
# If you set this to an empty list, the operator will not be given permission to read any additional secrets
# found in the cluster, and thus will only support a value of "false" in the Kiali CR setting
# spec.kiali_feature_flags.certificates_information_indicators.enabled.
```